### ISSUE TITLE
AttributeError: module 'faiss' has no attribute 'SearchParameters'

### ISSUE THREAD

**Issue by nk-fouque** [facebookresearch/faiss#2547]
# Summary

I am trying to find, for each vector of a matrix, the closest *among those above it*.
When trying to use parameters to search my index using a selector, I can't define SearchParameters and instead get the error
`AttributeError: module 'faiss' has no attribute 'SearchParameters'`

# Platform

OS: Ubuntu 20.04.5

Faiss version: 1.7.2

Installed from: `pip install faiss-gpu`

Running on:
- [ ] CPU
- [X] GPU

Interface: 
- [ ] C++
- [X] Python

# Reproduction instructions

```
index = faiss.IndexFlatL2(X_data.shape[1])
index.add(X_data.todense().astype('float32'))

closest_neighbor = []

for i in range(len(df2['text'])):
    sel = faiss.IDSelectorRange(0, i)
    params = faiss.SearchParameters(sel=sel)
    
    query=X_data[i].todense().astype('float32')
    k=1
    D,I = index.search(query,k, params=params)
    
    closest_neighbor += [D[0][0]]
```

**Comment by nk-fouque** [comment]
I have since realized that there is a much simpler way to do what I was trying to do (by only adding the vectors as I go), but I am still concerned by the fact that this error exists and that I couldn't create a SearchParameters if I actually needed to

**Comment by mdouze** [comment]
SearchParameters is available in the nighly builds (and in 1.7.3 when that comes out).
